lib/actions/action.js

```diff
@@ -0,0 +1,28 @@
+class Action {
```

Comment:
This class shares a lot of the same base traits as the Validator base class. I propose:

- Both Validator and Action extends `EventAware` which includes all relevant shared functions.
- `EventAware` can reside in `lib/`

Reply:
done